homeassistant/components/lametric/sensor.py

```diff
@@ -0,0 +1,87 @@
+"""Support for LaMetric sensors."""
+from __future__ import annotations
+
+from collections.abc import Callable
+from dataclasses import dataclass
+
+from demetriek import Device
+
+from homeassistant.components.sensor import (
+    SensorEntity,
+    SensorEntityDescription,
+    SensorStateClass,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import PERCENTAGE
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity import EntityCategory
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN
+from .coordinator import LaMetricDataUpdateCoordinator
+from .entity import LaMetricEntity
+
+
+@dataclass
+class LaMetricEntityDescriptionMixin:
+    """Mixin values for LaMetric entities."""
+
+    value_fn: Callable[[Device], int | None]
+
+
+@dataclass
+class LaMetricSensorEntityDescription(
+    SensorEntityDescription, LaMetricEntityDescriptionMixin
+):
+    """Class describing LaMetric sensor entities."""
+
+
+SENSORS = [
+    LaMetricSensorEntityDescription(
+        key="rssi",
+        name="Wi-Fi Signal",
```

Comment:
Since LaMetric entities set `has_entity_name` to True, shouldn't this conform to [the standard](https://developers.home-assistant.io/blog/2022/07/10/entity_naming)?

> Device, Area, and Entity names all start with a capital letter, the rest of the words are lower case (unless it's a word that represents a brand/name/abbreviation of course).

```suggestion
        name="Wi-Fi signal",
```